tests/testthat/error/test-slice-assign.txt

```diff
@@ -20,3 +20,24 @@ Error: Logical subscripts must match the size of the indexed input.
 i The input has size 32.
 x The subscript has size 2.
 
+
+must assign existing elements
+=============================
+
+> vec_assign(1:3, 5, 10)
+Error: Must assign existing elements.
+x Can't assign location 5.
+i There are only 3 elements.
+
+> vec_assign(1:3, "foo", 10)
+Error: Can't use character names to index an unnamed vector.
+
+> vec_slice(letters, -100) <- "foo"
+Error: Must negate existing elements.
```

Comment:
I initially misunderstood this as implying that existing elements must be negated. Maybe "Can't negate elements that don't exist" ?